ocean_lib/web3_internal/transactions.py

```diff
@@ -22,31 +19,7 @@ def send_ether(
     if not Web3.isChecksumAddress(to_address):
         to_address = Web3.toChecksumAddress(to_address)
 
-    receipt = accounts.at(from_wallet.address).transfer(to_address, amount)
-
-    return wait_for_transaction_status(receipt.txid, config["TRANSACTION_TIMEOUT"])
-
-
-def wait_for_transaction_status(txid: str, timeout: int):
-    receipt = TransactionReceipt(txid)
-
-    if timeout == 0:
-        return txid
-
-    start = time.time()
-    receipt = TransactionReceipt(txid)
-
-    if receipt.status.value == 1:
-        return txid
-
-    while time.time() - start > timeout:
-        receipt = TransactionReceipt(txid)
-        if receipt.status.value == 1:
-            return txid
-
-        time.sleep(0.2)
-
-    raise Exception("Transaction Timeout reached without successful status.")
+    return accounts.at(from_wallet.address).transfer(to_address, amount)
```

Comment:
I think it can be even simpler.

Rather than:
```python
accounts.at(from_wallet.address).transfer(to_address, amount)
```

It can be:
```python
from_wallet.transfer(to_address, amount)
```

Reply:
And then the question is: do we really even need `transactions.send_ether()` ? It's only used in three places in the tests, and those usages could easily be `from_wallet.transfer(to_address, amount)` instead.